be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.en.resx

```diff
@@ -156,6 +159,12 @@
   <data name="LastNameRequiredErrorMessage" xml:space="preserve">
     <value>Enter your last name</value>
   </data>
+  <data name="MarketingApprovalHeader" xml:space="preserve">
+    <value>Marketing form for people cooperating with ELTAP</value>
+  </data>
+  <data name="MarketingApprovalText" xml:space="preserve">
+    <value>If you would like to receive information from us about the current offer, current campaigns, introduced functionalities or changes, as well as other activities conducted by ELTAP, please complete the marketing consents below. Consents may be withdrawn at any time, and the withdrawal of consent does not affect actions taken before the date of its withdrawal.</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP